### DISCUSSION TITLE
客户端支持SteamOS吗？

### DISCUSSION THREAD

**Discussion by xskxzr** [v2fly/v2ray-core#1904] [Q&A]
RT，想买个Steam Deck，不知道支不支持v2ray

**Comment by AkinoKaede** [comment]
Steam OS is a Linux distribution, so you can run V2Ray on it.

**Comment by gylove1994** [comment]
可以，但是在游戏模式中似乎无法通过v2ray进行代理

**Reply by AkinoKaede** [reply]
Some software don't support proxy, you can use Transparent Proxy.

**Comment by gylove1994** [comment]
Thanks, I fix it.

秋のかえで ***@***.***>于2022年11月25日 周五19:16写道：
